source/features/toggle-files-button.css

```diff
@@ -4,15 +4,6 @@
 	padding: 10px 13px;
 }
 
-.rgh-files-hidden .commit-tease {
```

Comment:
`.commit-tease` seems to disappear in the new design.

This file is also touched by some other open PR, but I can't find it.